content/lessons/table_of_contents.json

```diff
@@ -1,36 +1,53 @@
 {
   "id": "table-of-contents",
-  "coolness": "very-cool",
-  "chapters": {
-    "one": {
```

Comment:
These "named entries" make no sense because we have to repeat the whole datastructure with these names in the GraphQL query for the TOC. This should be an array of arbitrary length that is evaluated dynamically and not expected to have hardcoded keys in it.